Sources/VGSCollectSDK/Core/APIClient.swift

```diff
@@ -145,8 +145,13 @@ class APIClient {
         // Add Headers
         let version = ProcessInfo.processInfo.operatingSystemVersion
         let versionString = "\(version.majorVersion).\(version.minorVersion).\(version.patchVersion)"
+
+				var trStatus = "default"
+				if !VGSAnalyticsClient.shared.shouldCollectAnalytics {
```

Comment:
could we make it one line:
let trStatus = VGSAnalyticsClient.shared.shouldCollectAnalytics ? "default" : "none"

Reply:
Done.